Program.cs

```diff
@@ -11,101 +49,96 @@ static Rectangle CellToRectangle(int row, int col, int size)
     }
 
     // trying to draw the apple shape? 
-    static Vector2 CellToCenter(int row, int col, int size)
+    static Vector2 CellToCenter(Cord appleCord, int size)
     {
         // Uhhh, we're converting thaaa indices of the cells into 
         // the centers of where they need to drawn (in the pixel world :o )
-        float centerX = size * (col + 0.5f);
-        float centerY = size * (row + 0.5f);
+        float centerX = size * (appleCord.Y + 0.5f);
+        float centerY = size * (appleCord.X + 0.5f);
```

Comment:
is this right? isn't it col == X and row == Y ? why are you centering X with Y and Y with X?